### PR TITLE
without-all for rotate-feedstock-tokens

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [x] Added a ``news`` entry

### CODE REVIEW

File: conda_smithy/cli.py [hunk 1]
```diff
@@ -902,6 +920,14 @@ class UpdateAnacondaToken(Subcommand):
         "update-binstar-token",
         "rotate-binstar-token",
     ]
+    ci_names = [
```
Comment (jaimergp) [inline conda_smithy/cli.py:923]: Better to have this as immutable if it's going to be shared as a class attribute.

**Comment by beckermr** [comment]
Once this is tested live let's go ahead and merge!

**Comment by isuruf** [comment]
I tested this PR on admin-requests with `register-feedstock-token` and it works as intended.

**Review by beckermr — APPROVED** [review]

**Comment by isuruf** [comment]
Thanks

**Pull request merged**